file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237
        throw new Error(y, { cause: u });
              ^

Error: [27-May-2025 04:08:21 UTC] PHP Deprecated:  Required parameter $attribute_name follows optional parameter $args in /wordpress/wp-content/plugins/navigation-du-lapin-blanc/navigation-du-lapin-blanc.php on line 646
[27-May-2025 04:08:22 UTC] PHP Deprecated:  Required parameter $attribute_name follows optional parameter $args in /wordpress/wp-content/plugins/navigation-du-lapin-blanc/navigation-du-lapin-blanc.php on line 646
[27-May-2025 04:08:29 UTC] PHP Deprecated:  Required parameter $attribute_name follows optional parameter $args in /wordpress/wp-content/plugins/navigation-du-lapin-blanc/navigation-du-lapin-blanc.php on line 646
[27-May-2025 04:08:30 UTC] PHP Deprecated:  Required parameter $attribute_name follows optional parameter $args in /wordpress/wp-content/plugins/navigation-du-lapin-blanc/navigation-du-lapin-blanc.php on line 646
[27-May-2025 04:08:39 UTC] PHP Deprecated:  Required parameter $attribute_name follows optional parameter $args in /wordpress/wp-content/plugins/navigation-du-lapin-blanc/navigation-du-lapin-blanc.php on line 646
[27-May-2025 04:08:40 UTC] PHP Deprecated:  Required parameter $attribute_name follows optional parameter $args in /wordpress/wp-content/plugins/navigation-du-lapin-blanc/navigation-du-lapin-blanc.php on line 646
[27-May-2025 04:08:49 UTC] PHP Warning:  include_once(/wordpress/wp-content/plugins/never-po/install.php): Failed to open stream: No such file or directory in /wordpress/wp-content/plugins/never-po/functions.php on line 18
[27-May-2025 04:08:49 UTC] PHP Warning:  include_once(): Failed opening '/wordpress/wp-content/plugins/never-po/install.php' for inclusion (include_path='.:') in /wordpress/wp-content/plugins/never-po/functions.php on line 18
[27-May-2025 04:08:58 UTC] PHP Warning:  include_once(/wordpress/wp-content/plugins/never-po/install.php): Failed to open stream: No such file or directory in /wordpress/wp-content/plugins/never-po/functions.php on line 18
[27-May-2025 04:08:58 UTC] PHP Warning:  include_once(): Failed opening '/wordpress/wp-content/plugins/never-po/install.php' for inclusion (include_path='.:') in /wordpress/wp-content/plugins/never-po/functions.php on line 18
[27-May-2025 04:09:07 UTC] PHP Warning:  include_once(/wordpress/wp-content/plugins/never-po/install.php): Failed to open stream: No such file or directory in /wordpress/wp-content/plugins/never-po/functions.php on line 18
[27-May-2025 04:09:07 UTC] PHP Warning:  include_once(): Failed opening '/wordpress/wp-content/plugins/never-po/install.php' for inclusion (include_path='.:') in /wordpress/wp-content/plugins/never-po/functions.php on line 18
[27-May-2025 04:12:45 UTC] PHP Warning:  Undefined variable $pagenow in /wordpress/wp-content/plugins/nav-menu-query-meta-box/nav-query-meta-box.php on line 14
[27-May-2025 04:12:53 UTC] PHP Warning:  Undefined variable $pagenow in /wordpress/wp-content/plugins/nav-menu-query-meta-box/nav-query-meta-box.php on line 14
[27-May-2025 04:13:01 UTC] PHP Warning:  Undefined variable $pagenow in /wordpress/wp-content/plugins/nav-menu-query-meta-box/nav-query-meta-box.php on line 14
[27-May-2025 04:13:45 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT blog_id FROM  ORDER BY domain ASC</p>
<p>Queries made or created this session were:</p>
<ol>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Failed to parse the MySQL query.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(606): WP_SQLite_Driver->new_driver_exception('Failed to parse...')
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(491): WP_SQLite_Driver->query('SELECT blog_id ...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(432): WP_SQLite_DB->_do_query('SELECT blog_id ...')
#3 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('SELECT blog_id ...')
#4 /wordpress/wp-content/plugins/network-summary/includes/class-network-summary.php(66): wpdb->get_col('SELECT blog_id ...')
#5 /wordpress/wp-content/plugins/network-summary/includes/class-network-summary.php(48): Network_Summary->get_sites()
#6 /wordpress/wp-includes/class-wp-hook.php(324): Network_Summary->activate(false)
#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_networ...', false)
#10 /internal/eval.php(20): activate_plugin('network-summary...')
#11 {main}</pre>
 for query SELECT blog_id FROM  ORDER BY domain ASC made by activate_plugin, do_action('activate_network-summary/network-summary.php'), WP_Hook->do_action, WP_Hook->apply_filters, Network_Summary->activate, Network_Summary->get_sites, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[27-May-2025 04:13:45 UTC] PHP Warning:  Trying to access array offset on value of type null in /wordpress/wp-content/plugins/network-summary/includes/class-site-category-repository.php on line 16
[27-May-2025 04:13:45 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>CREATE TABLE wp_site_categories (
			id BIGINT NOT NULL AUTO_INCREMENT,
			created TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
			name VARCHAR(55) NOT NULL,
			description VARCHAR(255) DEFAULT NULL,
			PRIMARY KEY id (id)
		)
		ENGINE ,
		DEFAULT COLLATE utf8_general_ci</p>
<p>Queries made or created this session were:</p>
<ol>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Failed to parse the MySQL query.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(606): WP_SQLite_Driver->new_driver_exception('Failed to parse...')
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(491): WP_SQLite_Driver->query('CREATE TABLE wp...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(432): WP_SQLite_DB->_do_query('CREATE TABLE wp...')
#3 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE wp...')
#4 /wordpress/wp-content/plugins/network-summary/includes/class-site-category-repository.php(29): dbDelta(Array)
#5 /wordpress/wp-content/plugins/network-summary/includes/class-network-summary.php(53): Site_Category_Repository->create_table()
#6 /wordpress/wp-includes/class-wp-hook.php(324): Network_Summary->activate(false)
#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_networ...', false)
#10 /internal/eval.php(20): activate_plugin('network-summary...')
#11 {main}</pre>
 for query CREATE TABLE wp_site_categories (
			id BIGINT NOT NULL AUTO_INCREMENT,
			created TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
			name VARCHAR(55) NOT NULL,
			description VARCHAR(255) DEFAULT NULL,
			PRIMARY KEY id (id)
		)
		ENGINE ,
		DEFAULT COLLATE utf8_general_ci made by activate_plugin, do_action('activate_network-summary/network-summary.php'), WP_Hook->do_action, WP_Hook->apply_filters, Network_Summary->activate, Site_Category_Repository->create_table, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[27-May-2025 04:13:45 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>CREATE TABLE wp_site_categories_relationships (
			blog_id BIGINT NOT NULL,
			category_id BIGINT NOT NULL,
			PRIMARY KEY id (blog_id, category_id),
			INDEX blog_ind (blog_id),
			INDEX category_ind (category_id),
			FOREIGN KEY (blog_id)
				REFERENCES (blog_id),
			FOREIGN KEY (category_id)
				REFERENCES wp_site_categories(id)
		)
		ENGINE ,
		DEFAULT COLLATE utf8_general_ci</p>
<p>Queries made or created this session were:</p>
<ol>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Failed to parse the MySQL query.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(606): WP_SQLite_Driver->new_driver_exception('Failed to parse...')
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(491): WP_SQLite_Driver->query('CREATE TABLE wp...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(432): WP_SQLite_DB->_do_query('CREATE TABLE wp...')
#3 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE wp...')
#4 /wordpress/wp-content/plugins/network-summary/includes/class-site-category-repository.php(45): dbDelta(Array)
#5 /wordpress/wp-content/plugins/network-summary/includes/class-network-summary.php(53): Site_Category_Repository->create_table()
#6 /wordpress/wp-includes/class-wp-hook.php(324): Network_Summary->activate(false)
#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_networ...', false)
#10 /internal/eval.php(20): activate_plugin('network-summary...')
#11 {main}</pre>
 for query CREATE TABLE wp_site_categories_relationships (
			blog_id BIGINT NOT NULL,
			category_id BIGINT NOT NULL,
			PRIMARY KEY id (blog_id, category_id),
			INDEX blog_ind (blog_id),
			INDEX category_ind (category_id),
			FOREIGN KEY (blog_id)
				REFERENCES (blog_id),
			FOREIGN KEY (category_id)
				REFERENCES wp_site_categories(id)
		)
		ENGINE ,
		DEFAULT COLLATE utf8_general_ci made by activate_plugin, do_action('activate_network-summary/network-summary.php'), WP_Hook->do_action, WP_Hook->apply_filters, Network_Summary->activate, Site_Category_Repository->create_table, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[27-May-2025 04:13:46 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/network-summary/includes/class-network-summary.php on line 197
[27-May-2025 04:13:54 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT blog_id FROM  ORDER BY domain ASC</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT blog_id FROM  ORDER BY domain ASC | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 near "ORDER": syntax error.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('SELECT blog_id ...')
#2 /wordpress/wp-content/plugins/network-summary/includes/class-network-summary.php(66): wpdb->get_col('SELECT blog_id ...')
#3 /wordpress/wp-content/plugins/network-summary/includes/class-network-summary.php(48): Network_Summary->get_sites()
#4 /wordpress/wp-includes/class-wp-hook.php(324): Network_Summary->activate(false)
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_networ...', false)
#8 /internal/eval.php(20): activate_plugin('network-summary...')
#9 {main}</pre>
 for query SELECT blog_id FROM  ORDER BY domain ASC made by activate_plugin, do_action('activate_network-summary/network-summary.php'), WP_Hook->do_action, WP_Hook->apply_filters, Network_Summary->activate, Network_Summary->get_sites, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[27-May-2025 04:13:54 UTC] PHP Warning:  Undefined array key "ENGINE" in /wordpress/wp-content/plugins/network-summary/includes/class-site-category-repository.php on line 16
[27-May-2025 04:13:55 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>CREATE TABLE wp_site_categories (
			id BIGINT NOT NULL AUTO_INCREMENT,
			created TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
			name VARCHAR(55) NOT NULL,
			description VARCHAR(255) DEFAULT NULL,
			PRIMARY KEY id (id)
		)
		ENGINE ,
		DEFAULT COLLATE utf8_general_ci</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: Cannot combine AUTOINCREMENT and multiple primary keys in SQLite.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE wp...')
#2 /wordpress/wp-content/plugins/network-summary/includes/class-site-category-repository.php(29): dbDelta(Array)
#3 /wordpress/wp-content/plugins/network-summary/includes/class-network-summary.php(53): Site_Category_Repository->create_table()
#4 /wordpress/wp-includes/class-wp-hook.php(324): Network_Summary->activate(false)
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_networ...', false)
#8 /internal/eval.php(20): activate_plugin('network-summary...')
#9 {main}</pre>
 for query CREATE TABLE wp_site_categories (
			id BIGINT NOT NULL AUTO_INCREMENT,
			created TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
			name VARCHAR(55) NOT NULL,
			description VARCHAR(255) DEFAULT NULL,
			PRIMARY KEY id (id)
		)
		ENGINE ,
		DEFAULT COLLATE utf8_general_ci made by activate_plugin, do_action('activate_network-summary/network-summary.php'), WP_Hook->do_action, WP_Hook->apply_filters, Network_Summary->activate, Site_Category_Repository->create_table, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[27-May-2025 04:13:55 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>CREATE TABLE wp_site_categories_relationships (
			blog_id BIGINT NOT NULL,
			category_id BIGINT NOT NULL,
			PRIMARY KEY id (blog_id, category_id),
			INDEX blog_ind (blog_id),
			INDEX category_ind (category_id),
			FOREIGN KEY (blog_id)
				REFERENCES (blog_id),
			FOREIGN KEY (category_id)
				REFERENCES wp_site_categories(id)
		)
		ENGINE ,
		DEFAULT COLLATE utf8_general_ci</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)
				VALUES (:table, :column, :datatype)
				ON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype
			 | parameters: wp_site_categories_relationships, blog_id, bigint</li>
<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)
				VALUES (:table, :column, :datatype)
				ON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype
			 | parameters: wp_site_categories_relationships, category_id, bigint</li>
<li>Executing: CREATE TABLE &quot;wp_site_categories_relationships&quot; (
&quot;blog_id&quot; integer NOT NULL ON CONFLICT REPLACE DEFAULT 0,
&quot;category_id&quot; integer NOT NULL ON CONFLICT REPLACE DEFAULT 0,
PRIMARY KEY (&quot;(&quot;, &quot;,&quot;, &quot;)&quot;)) | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 expressions prohibited in PRIMARY KEY and UNIQUE constraints.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE wp...')
#2 /wordpress/wp-content/plugins/network-summary/includes/class-site-category-repository.php(45): dbDelta(Array)
#3 /wordpress/wp-content/plugins/network-summary/includes/class-network-summary.php(53): Site_Category_Repository->create_table()
#4 /wordpress/wp-includes/class-wp-hook.php(324): Network_Summary->activate(false)
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_networ...', false)
#8 /internal/eval.php(20): activate_plugin('network-summary...')
#9 {main}</pre>
 for query CREATE TABLE wp_site_categories_relationships (
			blog_id BIGINT NOT NULL,
			category_id BIGINT NOT NULL,
			PRIMARY KEY id (blog_id, category_id),
			INDEX blog_ind (blog_id),
			INDEX category_ind (category_id),
			FOREIGN KEY (blog_id)
				REFERENCES (blog_id),
			FOREIGN KEY (category_id)
				REFERENCES wp_site_categories(id)
		)
		ENGINE ,
		DEFAULT COLLATE utf8_general_ci made by activate_plugin, do_action('activate_network-summary/network-summary.php'), WP_Hook->do_action, WP_Hook->apply_filters, Network_Summary->activate, Site_Category_Repository->create_table, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[27-May-2025 04:13:56 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/network-summary/includes/class-network-summary.php on line 197
[27-May-2025 04:14:03 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT blog_id FROM  ORDER BY domain ASC</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT blog_id FROM  ORDER BY domain ASC | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 near "ORDER": syntax error.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('SELECT blog_id ...')
#2 /wordpress/wp-content/plugins/network-summary/includes/class-network-summary.php(66): wpdb->get_col('SELECT blog_id ...')
#3 /wordpress/wp-content/plugins/network-summary/includes/class-network-summary.php(48): Network_Summary->get_sites()
#4 /wordpress/wp-includes/class-wp-hook.php(324): Network_Summary->activate(false)
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_networ...', false)
#8 /internal/eval.php(20): activate_plugin('network-summary...')
#9 {main}</pre>
 for query SELECT blog_id FROM  ORDER BY domain ASC made by activate_plugin, do_action('activate_network-summary/network-summary.php'), WP_Hook->do_action, WP_Hook->apply_filters, Network_Summary->activate, Network_Summary->get_sites, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[27-May-2025 04:14:03 UTC] PHP Warning:  Undefined array key "ENGINE" in /wordpress/wp-content/plugins/network-summary/includes/class-site-category-repository.php on line 16
[27-May-2025 04:14:03 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>CREATE TABLE wp_site_categories (
			id BIGINT NOT NULL AUTO_INCREMENT,
			created TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
			name VARCHAR(55) NOT NULL,
			description VARCHAR(255) DEFAULT NULL,
			PRIMARY KEY id (id)
		)
		ENGINE ,
		DEFAULT COLLATE utf8_general_ci</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: Cannot combine AUTOINCREMENT and multiple primary keys in SQLite.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE wp...')
#2 /wordpress/wp-content/plugins/network-summary/includes/class-site-category-repository.php(29): dbDelta(Array)
#3 /wordpress/wp-content/plugins/network-summary/includes/class-network-summary.php(53): Site_Category_Repository->create_table()
#4 /wordpress/wp-includes/class-wp-hook.php(324): Network_Summary->activate(false)
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_networ...', false)
#8 /internal/eval.php(20): activate_plugin('network-summary...')
#9 {main}</pre>
 for query CREATE TABLE wp_site_categories (
			id BIGINT NOT NULL AUTO_INCREMENT,
			created TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
			name VARCHAR(55) NOT NULL,
			description VARCHAR(255) DEFAULT NULL,
			PRIMARY KEY id (id)
		)
		ENGINE ,
		DEFAULT COLLATE utf8_general_ci made by activate_plugin, do_action('activate_network-summary/network-summary.php'), WP_Hook->do_action, WP_Hook->apply_filters, Network_Summary->activate, Site_Category_Repository->create_table, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[27-May-2025 04:14:04 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>CREATE TABLE wp_site_categories_relationships (
			blog_id BIGINT NOT NULL,
			category_id BIGINT NOT NULL,
			PRIMARY KEY id (blog_id, category_id),
			INDEX blog_ind (blog_id),
			INDEX category_ind (category_id),
			FOREIGN KEY (blog_id)
				REFERENCES (blog_id),
			FOREIGN KEY (category_id)
				REFERENCES wp_site_categories(id)
		)
		ENGINE ,
		DEFAULT COLLATE utf8_general_ci</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)
				VALUES (:table, :column, :datatype)
				ON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype
			 | parameters: wp_site_categories_relationships, blog_id, bigint</li>
<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)
				VALUES (:table, :column, :datatype)
				ON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype
			 | parameters: wp_site_categories_relationships, category_id, bigint</li>
<li>Executing: CREATE TABLE &quot;wp_site_categories_relationships&quot; (
&quot;blog_id&quot; integer NOT NULL ON CONFLICT REPLACE DEFAULT 0,
&quot;category_id&quot; integer NOT NULL ON CONFLICT REPLACE DEFAULT 0,
PRIMARY KEY (&quot;(&quot;, &quot;,&quot;, &quot;)&quot;)) | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 expressions prohibited in PRIMARY KEY and UNIQUE constraints.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE wp...')
#2 /wordpress/wp-content/plugins/network-summary/includes/class-site-category-repository.php(45): dbDelta(Array)
#3 /wordpress/wp-content/plugins/network-summary/includes/class-network-summary.php(53): Site_Category_Repository->create_table()
#4 /wordpress/wp-includes/class-wp-hook.php(324): Network_Summary->activate(false)
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_networ...', false)
#8 /internal/eval.php(20): activate_plugin('network-summary...')
#9 {main}</pre>
 for query CREATE TABLE wp_site_categories_relationships (
			blog_id BIGINT NOT NULL,
			category_id BIGINT NOT NULL,
			PRIMARY KEY id (blog_id, category_id),
			INDEX blog_ind (blog_id),
			INDEX category_ind (category_id),
			FOREIGN KEY (blog_id)
				REFERENCES (blog_id),
			FOREIGN KEY (category_id)
				REFERENCES wp_site_categories(id)
		)
		ENGINE ,
		DEFAULT COLLATE utf8_general_ci made by activate_plugin, do_action('activate_network-summary/network-summary.php'), WP_Hook->do_action, WP_Hook->apply_filters, Network_Summary->activate, Site_Category_Repository->create_table, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[27-May-2025 04:14:04 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/network-summary/includes/class-network-summary.php on line 197
[27-May-2025 04:15:07 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/news-list/news-ticker.php:223
Stack trace:
#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()
#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('news-list/news-...')
#2 /internal/eval.php(20): activate_plugin('news-list/news-...')
#3 {main}
  thrown in /wordpress/wp-content/plugins/news-list/news-ticker.php on line 223
[27-May-2025 04:15:14 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/news-list/news-ticker.php:223
Stack trace:
#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()
#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('news-list/news-...')
#2 /internal/eval.php(20): activate_plugin('news-list/news-...')
#3 {main}
  thrown in /wordpress/wp-content/plugins/news-list/news-ticker.php on line 223
[27-May-2025 04:15:22 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/news-list/news-ticker.php:223
Stack trace:
#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()
#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('news-list/news-...')
#2 /internal/eval.php(20): activate_plugin('news-list/news-...')
#3 {main}
  thrown in /wordpress/wp-content/plugins/news-list/news-ticker.php on line 223

    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)
    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
  [cause]: Error: Error when executing the blueprint step #1 ({"step":"installPlugin","pluginData":{"resource":"wordpress.org/plugins","slug":"news-list"},"options":{"activate":true}}) : PHP.run() failed with exit code 255 and the following output: 
      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)
      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)
      ... 2 lines matching cause stack trace ...
      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
    [cause]: PHPExecutionFailureError: PHP.run() failed with exit code 255 and the following output: 
        at PHP.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/universal/index.js:987:19)
        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
        at async wr (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:33:39)
        at async Object.gt [as installPlugin] (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:974:42)
        at async g (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12517:30)
        at async Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12446:23)
        at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)
        at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)
        at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)
        at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
      response: PHPResponse {
        httpStatusCode: 500,
        headers: {
          'x-powered-by': [ 'PHP/8.0.30-dev' ],
          'content-type': [ 'text/html; charset=UTF-8' ]
        },
        bytes: Uint8Array(11) [
          10, 32, 32, 32, 32,
          10, 10, 10,  9, 10,
          10
        ],
        exitCode: 255,
        errors: ''
      },
      source: 'request'
    }
  }
}

Node.js v23.11.0
